### ISSUE TITLE
Custom Row Layout

### ISSUE THREAD

**Issue by hbb20** [hbb20/AndroidCountryPicker#19]
Easy way to use different row layout

**Comment by gergely-szebenyi** [comment]
@hbb20 Is it possible to do it? I would like to change the image height, which is hardcoded to 14dp.

**Comment by hbb20** [comment]
No. It still doesn't have it. I was assuming that with jetpack compose, developers can use library provided data and build their own UI pretty easily. So I didn't prioritize it.

**Comment by gergely-szebenyi** [comment]
@hbb20 Thanks for this quick reply. I've solved it with overwriting cp_country_row.xml in my app. Not the most elagant solutions but it works.

If I have a custom dialog with an extra view, can I assign a clickListener to it to close the dialog without selection?

**Comment by gergely-szebenyi** [comment]
@hbb20 I see that clearSelectionButton does something similar (it closes the dialog) but also clears my previous selection (which is obvious by it's name). I'm currently trying to set an initial selection from code, not sure if it is possible

**Comment by gergely-szebenyi** [comment]
Managed to solve it.

**Comment by RahulSDeshpande** [comment]
@gergely-szebenyi That is great!! Can you please post your solution here?
